tests/Http/Controllers/Api/Entries/DeleteControllerMediaLocalTest.php

```
@@ -0,0 +1,597 @@
+            $response[0]->assertExactJson([
+                "data" => [
+                    "code" => "ec5_407",
+                    "deleted" => $chunkSize,
+                    "title" => "Chunk media deleted successfully."
+                ]
+            ]);
+            $this->assertCount($numOfEntries, Entry::where('project_id', $this->project->id)->get());
+
+            //assert media files are deleted, up to 1000
+            $photos = Storage::disk('entry_original')->files($this->project->ref);
+            $photos_thumbs = Storage::disk('entry_thumb')->files($this->project->ref);
```

PHPMD (2.15.0)

The variable $photos_thumbs is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)